__test__/shop.test.js

```diff
@@ -0,0 +1,89 @@
+var Shop = require('../src/shop');
+var Item = require('../src/item');
+
+describe("Gilded Rose", function() {
+  it('Name does not change after update', () => {
+    const gildedRose = new Shop([ new Item("foo", 0, 0) ]);
+    const items = gildedRose.updateQuality();
+    expect(items[0].name).toBe("foo");
+  });
+
+  it('Quality degrades twice as fast after sellIn is zero', () => {
```

Comment:
Try and use describe statements to group things like "Quality" maybe... 
It statements should start with a verb as a general rule!

Let me know if you need clarification. I think the tests in my workshop should help clarify this